### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by shahramk**
refactor
- set nr-labs-components dependency to be fetched from npm repository
- remove `helpModalOpen && `condition in JSX before modal component

### CODE REVIEW

File: nerdlets/github-about/.DS_Store
Comment (aso1124) [inline nerdlets/github-about/.DS_Store:1]: @shahramk This shouldn't be part of the commit - can you update .gitignore to exclude .DS_Store files?

**Comment by aso1124** [comment]
Looks good other than the DS_Store files

File: nerdlets/.DS_Store
[the remark this answers is not in the document]
Comment (shahramk) [reply nerdlets/.DS_Store:1]: main contains a `.gitignore` which already ignores `.DS_Store` files. 
There are 2 `.DS_Store` files in `main` from 5 years ago.
If I am not mistaken the `.DS_Store` entries in the commit will remove the existing `.DS_Store`.

[the remark this answers is not in the document]
Comment (shahramk) [reply nerdlets/.DS_Store:1]: main contains a `.gitignore` which already ignores `.DS_Store` files. 
There are 2 `.DS_Store` files in `main` from 5 years ago.
If I am not mistaken the `.DS_Store` entries in the commit will remove the existing `.DS_Store`.

[the remark this answers is not in the document]
Comment (shahramk) [reply nerdlets/.DS_Store:1]: main contains a `.gitignore` which already ignores `.DS_Store` files. 
There are 2 `.DS_Store` files in `main` from 5 years ago.
If I am not mistaken the `.DS_Store` entries in the commit will remove the existing `.DS_Store`.

**Review by aso1124 — APPROVED** [review]
👍
